core/src/test/scala/integration/kafka/api/EndToEndAuthorizationTest.scala

```diff
@@ -229,16 +296,60 @@ trait EndToEndAuthorizationTest extends IntegrationTestHarness with SaslSetup {
     //Produce records
     debug("Starting to send records")
     sendRecords(numRecords, tp)
-    //Consume records
+
+    //Deleting topic ACL without asking for console confirmation
```

Comment:
Why are you mentioning `without asking for console confirmation`? Isn't that the case for all ACLs calls here?